### ISSUE TITLE
Minor white roads are not visible in the daylight

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#3826]
Some users complain that it's hard to see white roads on the map in the bright sunlight. There are several possible solutions here:
1. Add black/gray borders to lines (it may slow down the rendering and consume more battery).
2. Highlight roads by using different colors in a different style (driving one? accessibility one?).

**Comment by ThomasAUS** [comment]
This is a brilliant and very important improvement to Organic Maps. I think it would be better to do the black border approach as otherwise the whole colour system will have to be changed.

The performance issue can be solved by having a tick box in the options to make this selectable.

If the white minor roads become solid black, what distinguishes them from railways and borders, then?

It would also be great to make the 4-lane major highways (autobahn, interstate highway) a different colour from regular main roads. At the moment they are all red.

The colours on viamichelin.com are an excellent example.

Thanks for your great work

**Comment by ThomasAUS** [comment]
I think version 1 is better as it would leave the colour scheme intact.

Maybe add an item in the options menu so that people can turn this functionality off if they are concerned about battery life.

Thanks and keep up the great work

PS. Would it be possible to highlight the big 4-lane roads (interstate highways / autobahn / autostrada / freeways) whatever you call them? That would be a mighty improvement to visibility as well

**Comment by biodranik** [comment]
> Would it be possible to highlight the big 4-lane roads (interstate highways / autobahn / autostrada / freeways) whatever you call them

Aren't they already highlighted? What is the difference with osm.org rendering?

**Comment by pastk** [comment]
Yes, at the moment motorways' color / rendering style is identical to `highway=trunk` and is not so much different from `highway=primary` (somewhat thicker and more intense color).

**Comment by pastk** [comment]
We can add borders on detailed zoom levels only, then performance won't be an issue and not issue with reducing readability on overview zooms when roads make a dense "web" sometimes.

**Comment by ThomasAUS** [comment]
That would be a great idea!

Am 30.11.2024 um 06:00 schrieb Konstantin Pastbin:
